pkg/default_validator.go

```diff
@@ -185,26 +196,182 @@ func convertPayloadToClaims(payload nutsJwt) (map[string]interface{}, error) {
 	return claims, nil
 }
 
-func convertClaimsToPayload(claims map[string]interface{}) (*nutsJwt, error) {
-	var (
-		jsonString []byte
-		err        error
-		payload    nutsJwt
-	)
+// StartSession starts an irma session.
+// This is mainly a wrapper around the irma.IrmaServer.StartSession
+func (v DefaultValidator) StartSession(request interface{}, handler irmaserver.SessionHandler) (*irma.Qr, string, error) {
+	return v.IrmaServer.StartSession(request, handler)
+}
 
-	if jsonString, err = json.Marshal(claims); err != nil {
-		return nil, fmt.Errorf("could not marshall payload: %w", err)
+// ParseAndValidateJwtBearerToken validates the jwt signature and returns the containing claims
+func (v DefaultValidator) ParseAndValidateJwtBearerToken(acString string) (*NutsJwtBearerToken, error) {
+	parser := &jwt.Parser{ValidMethods: []string{jwt.SigningMethodRS256.Name}}
```

Comment:
Is this a hardcoded sig-method or the allowed list of methods, in the latter some EC could be added?

Reply:
Indeed, since it should be a constant implementing NCSC recommendations. @reinkrul

Reply:
I agree. I'd suggest to implement it after we abstracted the key/algo types in crypto to support more than just RSA keys.

So just merge for now, then use crypto when it provides the list of preferred algos?